src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackV2VolumePluginTest.java

```diff
@@ -84,53 +81,73 @@ public void setUp() throws Exception {
         this.tokenDefault = new Token(FAKE_ACCESS_ID, tokenUser, new Date(), attributes);
     }
 
+    // test case: Check if the request in requestInstance() is executed properly with the right parameters.
     @Test
     public void testRequestInstance() throws IOException, FogbowManagerException {
+        // set up
         String url = FAKE_STORAGE_URL + V2_API_ENDPOINT + FAKE_TENANT_ID
                 + OpenStackV2VolumePlugin.SUFIX_ENDPOINT_VOLUMES;
         HttpUriRequest request = new HttpPost(url);
         addHeaders(request);
 
         JSONObject json = this.openStackV2VolumePlugin.generateJsonEntityToCreateInstance(FAKE_SIZE, FAKE_NAME);
 
-        doReturn(json).when(this.openStackV2VolumePlugin).generateJsonEntityToCreateInstance(anyString(), anyString());
+        Mockito.doReturn(json).when(this.openStackV2VolumePlugin).generateJsonEntityToCreateInstance(
```

Comment:
I do not understand why do we both call this.openStackV2VolumePlugin.generateJsonEntityToCreateInstance(FAKE_SIZE, FAKE_NAME);
in line 93 and mock the same call in line 95